gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/gateway_query.proto

```diff
@@ -107,8 +107,7 @@ message Expression {
 }
 
 message HealthExpression {
-  // Currently this is empty since there is nothing additional to provide.
-  // Presence of this expression itself indicates that the health of metric is required.
+  FunctionExpression function = 1;
```

Comment:
What is this? Because the baseline api is separate, we no longer need a message like this (i.e. it can remain defunct)

Reply:
Ok. Since health expression is already present i used, if function present inside health expression, then that function is eligible for baseline calculation.

Reply:
Isn't that function already represented by the parent aggregation? (moot point though, given the discussion about separating the APIs)